starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,44 @@
 // Names and Input
+var hacker1 = 'Daniel';
+console.log('The driver\'s name is '+hacker1);
+//var hacker2 = prompt('What is the name of the navigator?');
+console.log('The navigator\'s name is '+hacker2);
 
 
 //Conditionals
 
+if(hacker1.length>hacker2.length){
+  console.log('The Driver has the longest name, it has '+hacker1.length+' characters ');} else if(hacker1.length<hacker2.length){
+    console.log('Yo, navigator got the longest name, it has '+hacker2.length+' characters');
+  }else{
+    console.log('wow, you both got equally long names,'+hacker1.length+' characters!!');
+  }
+
+//loops
+// Exercise 6:
+var nameDef='';
+for(let i=0;i<hacker1.length;i++){
+  nameDef += hacker1[i].toUpperCase();
+  nameDef += ' ';
+}
+console.log('Driver\'s name: '+nameDef);
+
+//Exercise 7:
+var nameDef2='';
```

Comment:
Si utilizáis ```let``` usad ```const``` también en vez de ```var```. 
Como regla general: Uitlizar siempre ```const``` hasta que el código de error.